SnapshotTests/ScreenShareViewControllerLayoutTests.swift

```diff
@@ -0,0 +1,26 @@
+@testable import GliaWidgets
+import SnapshotTesting
+import XCTest
+
+class ScreenShareViewControllerLayoutTests: SnapshotTestCase {
+    func testScreenShareViewController() {
```

Comment:
Just to align with other tests
```suggestion
    func test_screenShareViewController() {
```

Reply:
It makes sense, for sure, but I think we will not benefit from changing it now, since it will require regeneration of references. It was like that in VoiceOver tests, that is why it is kept like that. The purpose of this PR is to keep differences at minimum in layout. I will create separate ticket for renaming VoiceOver test files, because some of them are missing `VoiceOverTests` suffix and address this suggestion as well.

Reply:
Oh, I see. You are right